spi/control-plane/transfer-spi/src/main/java/org/eclipse/edc/connector/transfer/spi/types/protocol/TransferSuspensionMessage.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.transfer.spi.types.protocol;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The {@link TransferSuspensionMessage} is sent by the provider or consumer at any point except a terminal state to
+ * indicate the data transfer process should be suspended. If the termination was due to an
+ * error, the sender may include error information.
+ */
+public class TransferSuspensionMessage extends TransferRemoteMessage {
+
+    private String code;
+    private List<Object> reason = new ArrayList<>();
```

Comment:
is `Object` mandated by DSP? if not, I would suggest to just use `String`.

Reply:
the spec shows an [example where there's an empty object](https://github.com/International-Data-Spaces-Association/ids-specification/blob/main/transfer/message/example/transfer-suspension-message.json), in fact every `reason` item could be anything, leaving `Object` will permit to handle different shapes for the errors